libs/wingsdk/src/math/math.ts

```diff
@@ -0,0 +1,145 @@
+import { Code, InflightClient } from "../core";
+
+/**
+ * Utility class for mathematical operations.
+ */
+export class Util {
+  /**
+   * The mathematical constant representing the ratio of a circle's circumference to its diameter.
+   */
+  public static readonly PI = 3.141592653589793;
```

Comment:
Life advice: whenever possible, don't calculate PI on your own:

```suggestion
  public static readonly PI = Math.PI;
```